dizoo/mujoco/config/halfcheetah_bdq_config.py

```diff
@@ -22,7 +22,6 @@
             action_bins_per_branch=2,  # mean the action shape is 6, 2 discrete actions for each action dimension
```

Comment:
why modify this

Reply:
It may be modified by format.sh, will I need to change it back?